ethicml/algorithms/inprocess/agarwal_reductions.py

```diff
@@ -11,13 +11,17 @@
 
 __all__ = ["Agarwal"]
 
+from .svm import KernelType
 
 VALID_FAIRNESS: Set[FairnessType] = {"DP", "EqOd"}
 VALID_MODELS: Set[ClassifierType] = {"LR", "SVM"}
 
 
 class Agarwal(InAlgorithmAsync):
-    """Agarwal class."""
+    """Agarwal class.
+
+    A wrapper around the Exponentiated Gradient method documented [here](https://fairlearn.org/v0.7.0/api_reference/fairlearn.reductions.html#fairlearn.reductions.ExponentiatedGradient).
```

Comment:
That's unfortunately not how links are done in ReST. [It's a little more complicated.](https://docutils.sourceforge.io/docs/user/rst/quickref.html#hyperlink-targets)

Reply:
I think it's
```suggestion
    A wrapper around the Exponentiated Gradient method documented `here <https://fairlearn.org/v0.7.0/api_reference/fairlearn.reductions.html#fairlearn.reductions.ExponentiatedGradient>`_.
```